I write the release note for this 27th of June (#14472)
ipython/ipython: docs/source/whatsnew/version8.rst

============
 8.x Series
============
.. _version 8.26:

IPython 8.26
============

> Hey, the release of IPython for this month are here! (I know you were waiting for them)

Reviewer: I thing there is a single release, is it's *is* not *are* 

```suggestion
Hey, the release of IPython for this month is here! (I know you were waiting for it)
```
